scripts/benchmark_bolt.py

```
@@ -0,0 +1,80 @@
+        "CREATE TABLE crew_availability (id INTEGER PRIMARY KEY AUTOINCREMENT, crew_id INTEGER NOT NULL, start_time DATETIME NOT NULL, end_time DATETIME NOT NULL)"
+    )
+    c.execute(
+        "CREATE TABLE appliance (id INTEGER PRIMARY KEY AUTOINCREMENT, name TEXT NOT NULL UNIQUE)"
+    )
+    c.execute(
+        "CREATE TABLE appliance_availability (id INTEGER PRIMARY KEY AUTOINCREMENT, appliance_id INTEGER NOT NULL, start_time DATETIME NOT NULL, end_time DATETIME NOT NULL)"
+    )
+
+    crew_data = []
+    for i in range(num_crew):
+        crew_data.append((f"Crew {i:03d}", "FFC", "BA LGV TTR", "42"))
```

Ruff (0.15.7)

[warning] Use a list comprehension to create a transformed list

(PERF401)